allennlp/common/testing/test_case.py

```diff
@@ -28,6 +28,7 @@ def setUp(self):
         logging.getLogger('allennlp.common.params').disabled = True
         logging.getLogger('allennlp.nn.initializers').disabled = True
         logging.getLogger('allennlp.modules.token_embedders.embedding').setLevel(logging.INFO)
+        logging.getLogger('urllib3.connectionpool').disabled = True
```

Comment:
This is used for checking etag stuff, I think, and when you set the logging level to debug with pytest, these messages start showing up.  Pretty sure we don't need them, but if anyone sees a reason to, we can remove this line.